lib/hydra/works/vocab/works_terms.rb

```diff
@@ -0,0 +1,19 @@
+require 'rdf'
+module RDFVocabularies
+  class WorksTerms < RDF::Vocabulary("http://hydra.org/works/models#")
```

Comment:
Please switch to `http://projecthydra.org/ns/works/models#` . hydra.org is something else entirely.